.travis.yml

```diff
@@ -1,23 +1,20 @@
-language: generic
 dist: xenial
 sudo: required
 
-matrix:
-  include:
 
-  - language: python
-    env:
-    - JOBNAME=UNIT_TESTS
+language: python
+python:
+  - 3.7
+  - 3.8
+  #- 3.9
 
-    python:
-    - "3.8"
+install:
+  - python --version
+  - sudo apt-get install -y sqlcipher
+  - sudo apt-get install -y libsqlcipher-dev
+  - pip3 install -r requirements.txt
+  - pip3 install dpytest==0.0.20
+  - pip3 install discord.py==1.3.4
```

Comment:
Custom versions of dpytest and discord.py should no longer be needed from #149

Reply:
Just need one of them to be merged to master, #149 was merged into #134, not master